.vib/gitea/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,24 @@
+const COMMAND_DELAY = 2000;
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, COMMAND_DELAY);
+    });
+  });
+}
+
+Cypress.Commands.add(
+  'login',
+  (username = Cypress.env('username'), password = Cypress.env('password')) => {
+    cy.visit('/user/login');
+    cy.get('input[name="user_name"]').should('be.enabled').type(username);
+    cy.get('input[type="password"]').should('be.enabled').type(password);
```

Comment:
It shouldn't be necessary to check if these inputs are enabled or not.

```suggestion
    cy.get('[name="user_name"]').type(username);
    cy.get('[type="password"]').type(password);
```